Lib/pydoc.py

```diff
@@ -1254,6 +1254,24 @@ def makename(c, m=object.__module__):
                 push('    ' + makename(base))
             push('')
 
+        # List the built-in subclasses, if any:
+        subclasses = sorted(
+            (str(cls.__name__) for cls in object.__subclasses__() \
```

Comment:
Backslash not needed here

Reply:
Oh, nice catch! Fixed this :)

Thank you!